core/modules/qproc/testQueryAna.h

```diff
@@ -74,30 +75,30 @@ void testParse(SelectParser::Ptr p) {
 * list will be loaded
 *
 */
-std::shared_ptr<QuerySession> prepareTestQuerySession(QuerySession::Test& t,
-                                          std::string const& stmt,
-                                          std::string const& expectedErr="") {
+std::shared_ptr<QuerySession> buildQuerySession(QuerySession::Test& t,
+                                                std::string const& stmt,
+                                                std::string const& expectedErr = "") {
```

Comment:
I believe in many other places we do not add spaces when specifying default value.